executors/src/eoa/error_classifier.rs

```diff
@@ -204,6 +211,14 @@ impl EoaErrorMapper {
                 retry_delay: None,
             },
 
+            EoaExecutionError::ThirdwebSupportError { .. } => RecoveryStrategy {
+                queue_confirmation: false,
+                recycle_nonce: false,
+                needs_resync: false,
+                retryable: true,
+                retry_delay: Some(Duration::from_secs(1)), // Short delay for retry in place
```

Comment:
How many retries do we do here? Forever?

Reply:
this file isn't even used anywhere tbh, needs to be trimmed out. error handling is elsewhere